chore: Updated release date for Node 16, and removed unused `@newrelic/next` (#18216)
newrelic/docs-website: src/content/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent.mdx

3. Use the command `npm install newrelic` for each application you want to monitor. If your app is using [one of these Apollo Server modules](/docs/apm/agents/nodejs-agent/extend-your-instrumentation/apollo-server-plugin-nodejs/) install our Apollo plugin with `npm install @newrelic/apollo-server-plugin`. [More details about using `@newrelic/apollo-server-plugin` can be found here](https://github.com/newrelic/newrelic-node-apollo-server-plugin/tree/main#readme).

  <Callout variant="important">

>     If you're using Next.js, here is an [example Next.js app](https://github.com/newrelic/newrelic-node-nextjs#example-project)

Reviewer: ```suggestion
    If you're using Next.js, see [this example of a Next.js app](https://github.com/newrelic/newrelic-node-nextjs#example-project)
```
